Ronin.rte/Actors/Infantry/RoninLight/Hair.lua

```diff
@@ -3,4 +3,7 @@ function Update(self)
 	local gravity = self.Vel - SceneMan.GlobalAcc * rte.PxTravelledPerFrame;
 	self.RotAngle = gravity.AbsRadAngle;
 	self.Frame = gravity.Magnitude > 5 and 0 or 1;
+	if not IsActor(self:GetRootParent()) then
```

Comment:
You should be able to accomplish this by setting the INI property `DeleteWhenRemovedFromParent = 1`

Reply:
It's actually more complicated than that - this attachable (and other non-GetsHitByMOs attachables, I think) bugs out for some reason when the head is separated from the body, causing the head to float around randomly. This is (hopefully) just a temporary workaround for until this gets fixed. My bad for not having put an issue for it yet.

Reply:
Ohhhh gotcha. Yeah, I have seen that happen in the past, didn't realize this was to fix that but glad you did. And yeah, please put in an issue sometime.